PromoExchange/batman
# Facebook company store

@PromoExchange/engineers 

This PR is to introduce a company store builder for Facebook. The result of this merge should be a fully operating store with a wine glass product ready for purchase.

🏁 (To Do)
- [x] Add facebook logo to S3
- [x] Upload facebook data to S3
- [ ] Write rake task to load the Facebook company store

📋 (Test Plan)
1. Navigate to `http://px-batman-stage-pr-698.herokuapp.com/company_store/facebook`
2. Confirm you can purchase a bundle of wine glasses successfully. Confirm emails are sent to mailtrap correctly.
